### DISCUSSION TITLE
Toolbar does not align on top of the selected block when scrolling

### DISCUSSION THREAD

**Discussion by esha-fatima** [GrapesJS/grapesjs#5323] [Q&A]
The entire badge showing edit options for a selected element on canvas does not align properly while scrolling. It seems that this is happening due to some changes made in [this](https://github.com/GrapesJS/grapesjs/commit/b4f17561ce0da65f6eeeafe12e0dc4ba81ca2c01) commit.

https://github.com/GrapesJS/grapesjs/assets/85702449/4fd34a2d-2da2-4131-9a21-8ed89898bac7

**Comment by artf** [comment]
Are you able to provide a reproducible demo?
